pkg/kademlia/testrouting/testrouting.go

```diff
@@ -0,0 +1,299 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package testrouting
+
+import (
+	"sort"
+	"sync"
+	"time"
+
+	"storj.io/storj/pkg/dht"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/storage"
+)
+
+type nodeData struct {
+	node      *pb.Node
+	ordering  int64
+	timestamp time.Time
```

Comment:
Maybe, `lastConnected`, `updated` or `lastUpdated` or something similar?

Reply:
lastUpdated is good